extensions/applicationinsights-angularplugin-js/README.md

```diff
@@ -1,3 +1,4 @@
+> Note: Angular Plugin is moved to its own repo [here](https://github.com/microsoft/applicationinsights-angularplugin-js). This Plugin is deprecated. Please refer to the new repo for usage instructions, and open issue there.
```

Comment:
It probably better to phrase as moved as the plugin itself is not deprecated, just where the code lives :smile:

Reply:
That makes a lot sense... XD 
What do you think of the update? Thanks.